docs/examples/classification.ipynb

```diff
@@ -0,0 +1,984 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Classification on Nursery Dataset\n",
+    "This notebook demonstrates model diagnostics on the nursery dataset https://www.openml.org/d/43938, originally https://doi.org/10.24432/C5P88W.\n",
+    "\n",
+    "Our modelling goal is to estimate the conditional probability of $Y=1$, i.e. $P(Y=1|X)$, where $X$ are the features and $Y$ is defined as\n",
+    "$$\n",
+    "Y\n",
+    "= \n",
+    "\\begin{cases}\n",
+    "1 \\text{ if class} \\in \\text{\\{priority, special priority\\}}\\\\\n",
+    "0 \\text{ if otherwise }\n",
```

Comment:
Could you add in the text the ordering of all the classes in the dataset?

Reply:
How about c6fae20b6eba637d01a83b628e08cf220af54068 ?